Update glossary.md (#171)
GFDRR/rdl-standard: docs/glossary.md

# Glossary of terms

In this section, you will find definitions of key concepts behind the RDLS including what is a disaster, how to reduce disaster risk through disaster risk management and disaster risk assessment and what are the different types of risk data needed.

## Disaster risk

Disaster risk is the potential loss of life, injury, or destroyed or damaged assets which could occur to a system, society or community in a specific period of time, determined probabilistically as a function of hazard, exposure, vulnerability and coping capacity.

Disaster risk comprises different types of potential impacts and losses (direct and indirect, tangible and intangible) which are sometimes difficult to quantify. Nevertheless, with knowledge of the prevailing hazards, the patterns of population and socio-economic development and their tendency to suffer impacts, disaster risks can be assessed and mapped, at least in broad terms.

Source: [UNDRR Terminology (Disaster Risk)](https://www.undrr.org/terminology/disaster-risk)

## Climate and disaster risk assessment

> A qualitative or quantitative approach to determine the nature and extent of climate disaster risk by analysing potential hazards and evaluating existing conditions of exposure and vulnerability that together could harm people, property, services, livelihoods and the environment on which they depend.

Reviewer: ```suggestion
A qualitative or quantitative approach to determine the nature and extent of climate [disaster risk](#disaster-risk) by analysing potential [hazards](#hazard) and evaluating existing conditions of [exposure](#exposure) and [vulnerability](vulnerability) that together could harm people, property, services, livelihoods and the environment on which they depend.
```

Just adding in inter page links to make it extra clear how this definition brings together the other definitions in the page.

> An impact or risk model (also damage model; loss model) is used to combine hazard, exposure and vulnerability components to calculate the impact or single or multiple events.

Reviewer: ```suggestion
An impact or risk model (also damage model; loss model) is used to combine hazard, exposure and vulnerability components to calculate the [impact](#disaster-impact) or single or multiple events.
```
I've not linked to the H, E or V definitions here as this definition is referring to the datasets for these components rather than the concept of them.

Source: [UNDRR Terminology (Disaster Risk Information)](https://www.undrr.org/terminology/disaster-risk-information)

## Disaster risk management

Disaster risk management is the application of disaster risk reduction policies and strategies to prevent new disaster risk, reduce existing disaster risk and manage residual risk, contributing to the strengthening of resilience and reduction of disaster losses.

Source: [UNDRR Terminology (Disaster Risk Management)](https://www.undrr.org/terminology/disaster-risk-management)

## Hazard

A hazard is a process or phenomenon that may cause loss of life, injury or other health impacts, property damage, social and economic disruption or environmental degradation.  Hazards may be single, sequential or combined in their origin and effects. Each hazard is characterized by its location, intensity or magnitude, frequency and probability. A hazardous event is the manifestation of a hazard in a particular place during a particular period of time.

> Hazards can have a natural or anthropogenic origin. Natural hazards are associated with natural processes and phenomena (geological, geophysical and hydrometeorological).

Reviewer: ```suggestion
Hazards can have a natural or anthropogenic origin. Natural hazards are associated with natural processes and phenomena (e.g. geological, geophysical, hydrometeorological).
```

Added the e.g. as this isn't a complete list of natural processes and phenomena

## Exposure

The situation of people, infrastructure, housing, production capacities and other tangible human assets located in hazard-prone areas. Measures of exposure can include the number of people or types of assets in an area. These can be combined with the specific vulnerability and capacity of the exposed elements to any particular hazard to estimate the quantitative risks associated with that hazard in the area of interest

Source: [UNDRR Terminology (Exposure)](https://www.undrr.org/terminology/exposure)

## Vulnerability

Vulnerability is the conditions determined by physical, social, economic and environmental factors or processes which increase the susceptibility of an individual, a community, assets or systems to the impacts of hazards. The inverse of vulnerability is defined as coping capacity, which is the combination of all the strengths, attributes and resources available within an organization, community or society to manage and reduce disaster risks and strengthen resilience.

Source: [UNDRR Terminology (Vulnerability)](https://www.undrr.org/terminology/vulnerability)

## Disaster Impact

> The effect of a disaster on people, buildings and society is generally referred to as the impact. The impact of disasters can generate losses, but also gains for some people and economies; for instance, the demand for construction materials and expertise following a disaster. It is, therefore, necessary to think of impact in terms of both gains and losses.

Reviewer: ```suggestion
The effect of a disaster on people, buildings and society is generally referred to as the impact. The impact of disasters can generate [losses](#loss), but also gains for some people and economies; for instance, the demand for construction materials and expertise following a disaster. It is, therefore, necessary to think of impact in terms of both gains and losses.
```

Source: Adapted from ([UNDRR](https://www.undrr.org/terminology/disaster))

## Loss

> Losses are a measure of this impact in the form of damage or destruction caused by a disaster (observed or modelled). This measure may be expressed as monetary (e.g. cost of reconstruction) or non-monetary (e.g. hectares of crop loss, loss of life). Losses can be direct or indirect. 

Reviewer: ```suggestion
Losses are a measure of [disaster impact](#disaster-impact) in the form of damage or destruction caused by a disaster (observed or modelled). This measure may be expressed as monetary (e.g. cost of reconstruction) or non-monetary (e.g. hectares of crop loss, loss of life).
```

> The total economic impact consists of a monetary estimate of both direct disaster loss and indirect disaster loss. _Direct disaster losses_ refer to directly quantifiable losses such as the number of people killed and the damage to buildings, infrastructure and natural resources. _Indirect disaster losses_ refer to wider and longer-term impacts arising from the direct disruptions to the flow of goods and services as a result of a disaster. This includes declines in output or revenue, increases in food prices, and other longer-term social and economic effects on education, health, productivity, the well-being of people or the macroeconomy. 

Reviewer: ```suggestion
Losses can be direct or indirect with the total economic impact consisting of a monetary estimate of both direct disaster loss and indirect disaster loss.

 ### Direct disaster loss
Directly quantifiable losses such as the number of people killed and the damage to buildings, infrastructure and natural resources. 
 
 ### Indirect disaster loss
Wider and longer-term impacts arising from direct disruptions to the flow of goods and services as a result of a disaster. This includes declines in output or revenue, increases in food prices, and other longer-term social and economic effects on education, health, productivity, the well-being of people or the macroeconomy. 
```
Separating these out into sub-headings makes it easier to parse.

Source: Adapted from ([UNDRR](https://www.preventionweb.net/understanding-disaster-risk/key-concepts/direct-indirect-losses))
and [UNDRR Terminology (Economic Loss)](https://www.undrr.org/terminology/economic-loss)

## People affected

> People who have been exposed and impacted, either directly or indirectly, by a hazardous event. Directly affected are those who have suffered injury, illness or other health effects; who were evacuated, displaced, relocated or have suffered direct damage to their livelihoods, economic, physical, social, cultural and environmental assets. Indirectly affected are people who have suffered consequences, other than or in addition to direct effects, over time, due to disruption or changes in the economy, critical infrastructure, basic services, commerce or work, or social, health and psychological consequences.

Reviewer: ```suggestion
People who have been exposed and impacted, either directly or indirectly, by a hazardous event. 

Directly affected are those who:
* have suffered injury, illness or other health effects,
* were evacuated, displaced, or relocated,
* have suffered direct damage to their livelihoods, economic, physical, social, cultural and environmental assets. 

Indirectly affected are people who have suffered consequences, other than or in addition to direct effects, over time, due to:
* disruption or changes in the economy, critical infrastructure, basic services, commerce or work,
* social, health or psychological consequences.
```

Using bullet pointed lists just makes it a bit easier to parse these lists of lists.
